include/sway/config.h

```diff
@@ -333,8 +333,11 @@ enum edge_border_types {
 	E_VERTICAL,     /**< hide vertical edge borders */
 	E_HORIZONTAL,   /**< hide horizontal edge borders */
 	E_BOTH,		/**< hide vertical and horizontal edge borders */
-	E_SMART, /**< hide both if precisely one window is present in workspace */
-	E_SMART_NO_GAPS, /**< hide both if one window and gaps to edge is zero */
+};
+enum edge_border_smart_types {
+	E_DUMB,
```

Comment:
nit: can you change the prefix of these so they differ from the other enum? Maybe `ESMART_{OFF,ON,NO_GAPS|`?